executorlib/task_scheduler/interactive/blockallocation.py

```
@@ -228,9 +228,11 @@ def _execute_multiple_tasks(
         worker_id=worker_id,
     )
     if init_function is not None:
-        interface.send_dict(
+        result = interface.send_and_receive_dict(
             input_dict={"init": True, "fn": init_function, "args": (), "kwargs": {}}
         )
+        if "error" in result and result["error"]:
```

Ruff (0.12.2)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)